osbuild/util/runners.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/libexec/platform-python
+
+import os.path
+import pathlib
+import platform
+import shutil
+import subprocess
+import sys
+
+
+def ldconfig(*args):
+    # ld.so.conf must exist, or `ldconfig` throws a warning
+    subprocess.run(["touch", "/etc/ld.so.conf"], check=True)
+
+    with open("/etc/ld.so.conf", "w", encoding="utf8") as f:
```

Comment:
Unless I'm missing something here this will override /etc/ld.so.conf with empty content in e.g. the fedora runner. This is maybe fine but it seems also slightly dangerous, maybe only do it if `len(args) > 0`?

Reply:
You are right, I missed that part, will fix it in next push